---
layout: archive
title: "Publications"
permalink: /cv/
author_profile: true
redirect_from:

---

{% include base_path %}

## Rapid morphological divergence following a human-mediated introduction: the role of drift and directional selection
*Heredity* **124**, 535–549 (2020)

**Authors:** Sendell-Price AT, Ruegg KC, Clegg SM.

Theory predicts that when populations are established by few individuals, random founder effects can facilitate rapid phenotypic divergence even in the absence of selective processes. However, empirical evidence from historically documented colonisations suggest that, in most cases, drift alone is not sufficient to explain the rate of morphological divergence. Here, using the human-mediated introduction of the silvereye (*Zosterops lateralis*) to French Polynesia, which represents a potentially extreme example of population founding, we reassess the potential for morphological shifts to arise via drift alone. Despite only 80 years of separation from their New Zealand ancestors, French Polynesian silvereyes displayed significant changes in body and bill size and shape, most of which could be accounted for by drift, without the need to invoke selection. However, signatures of selection at genes previously identified as candidates for bill size and body shape differences in a range of bird species, also suggests a role for selective processes in driving morphological shifts within this population. Twenty-four SNPs in our RAD-Seq dataset were also found to be strongly associated with phenotypic variation. Hence, even under population founding extremes, when it is difficult to reject drift as the sole mechanism based on rate tests of phenotypic shifts, the additional role of divergent natural selection in novel environments can be revealed at the level of the genome.

Download here: [link](https://asendellprice.github.io/files/Sendell-Price_et_al_2020_Heredity.pdf)

Link to podcast: [Drift and selection in paradise](https://play.acast.com/s/heredity-podcast-naturecom-science-feeds/driftandselectioninparadise)

___

## The genomic landscape of divergence across the speciation continuum in island-colonising silvereyes (*Zosterops lateralis*)
*G3: Genes, Genomics, Genetics* **10**, 3147-3163 (2020)

**Authors:**  Sendell-Price AT,  Ruegg KC, Anderson EC,  Quilodrán CS,  Van Doren BM,  Underwood VL,  Coulson T, Clegg SM.

Inferring the evolutionary dynamics at play during the process of speciation by analyzing the genomic landscape of divergence is a major pursuit in population genomics. However, empirical assessments of genomic landscapes under varying evolutionary scenarios that are known a priori are few, thereby limiting our ability to achieve this goal. Here we combine RAD-sequencing and individual-based simulations to evaluate the genomic landscape of divergence in the silvereye (*Zosterops lateralis*). Using pairwise comparisons that differ in divergence timeframe and the presence or absence of gene flow, we document how genomic patterns accumulate along the speciation continuum. In contrast to previous predictions, our results provide limited support for the idea that divergence accumulates around loci under divergent selection or that genomic islands widen with time. While a small number of genomic islands were found in populations diverging with and without gene flow, in few cases were SNPs putatively under selection tightly associated with genomic islands. The transition from localized to genome-wide levels of divergence was captured using individual-based simulations that considered only neutral processes. Our results challenge the ubiquity of existing verbal models that explain the accumulation of genomic differences across the speciation continuum and instead support the idea that divergence both within and outside of genomic islands is important during the speciation process.

Download here: [link](https://asendellprice.github.io/files/Sendell-Price_et_al_2020_G3.pdf)

___

## Digest: The role of linkage in mimicking “magic traits”
*Evolution* **74**, 2539-2540 (2020)

**Authors:**  Estandia A, Sendell-Price AT.

Can divergence in a mating trait increase local adaption by increasing ecological divergence? Servedio and Bürger propose that “pseudomagic traits,” tightly linked complexes consisting of an ecological locus under divergent selection and a locus acting as a mating cue, can effectively mimic pleiotropy. Such pseudomagic traits can form even when linkage between ecological and mating loci is limited.

Download here: [link](https://asendellprice.github.io/files/Estandia_Sendell-Price_2020_Evol.pdf)

___

## The multiple population genetic and demographic routes to islands of genomic divergence
*Methods in Ecology and Evolution* **11**, 6-21 (2020)

**Authors:**  Quilodrán CS, Ruegg KC, Sendell‐Price AT, Anderson EC, Coulson T, Clegg SM.

The way that organisms diverge into reproductively isolated species is a major question in biology. The recent accumulation of genomic data provides promising opportunities to understand the genomic landscape of divergence, which describes the distribution of differences across genomes. Genomic areas of unusually high differentiation have been called genomic islands of divergence. Their formation has been attributed to a variety of mechanisms, but a prominent hypothesis is that they result from divergent selection over a small portion of the genome, with surrounding areas homogenized by gene flow. Such islands have often been interpreted as being associated with divergence with gene flow. However, other mechanisms related to genomic structure and population history can also contribute to the formation of genomic islands of divergence. We currently lack a quantitative framework to examine the dynamics of genomic landscapes under the complex and nuanced conditions that are found in natural systems. Here, we develop an individual‐based simulation to explore the dynamics of diverging genomes under various scenarios of gene flow, selection and genotype–phenotype maps. Our modelling results are consistent with empirical observations demonstrating the formation of genomic islands under genetic isolation. Importantly, we have quantified the range of conditions that produce genomic islands. We demonstrate that the initial level of genetic diversity, drift, time since divergence, linkage disequilibrium, strength of selection and gene flow are all important factors that can influence the formation of genomic islands. Because the accumulation of genomic differentiation over time tends to erode the signal of genomic islands, genomic islands are more likely to be observed in recently divergent taxa, although not all recently diverged taxa will necessarily exhibit islands of genomic divergence. Gene flow primarily slows the swamping of islands of divergence with time. By using this framework, further studies may explore the relative influence of particular suites of events that contribute to the emergence of genomic islands under sympatric, parapatric and allopatric conditions. This approach represents a novel tool to explore quantitative expectations of the speciation process, and should prove useful in elucidating past and projecting future genomic evolution of any taxa.

Download here: [link](https://asendellprice.github.io/files/Quilodran_et_al_2020_MEE.pdf)
